### ISSUE TITLE
About Overfitting

### ISSUE THREAD

**Issue by yangheart** [lululxvi/deepxde#154]
Dear Dr. Lu,

I am using deepxde to solve an inverse problem in which the solution data is known and the boundary condition is also known, but some of the coefficients of PDE are unknown. 

The results matched the real coefficients very well. In addition, the solution also matched the solution data very well. However, when I did predictions, the results did not look good. So I am wondering if there exists overfitting in solving the inverse problem.  

In the case of overfitting,  I added dropout, but the training loss did not decrease enough to be below the tolerance, which leads to bad coefficient predictions. Could you please give me some suggestions?

Thank you so much!

**Comment by lululxvi** [comment]
I didn't get what you mean by "The results matched the real coefficients very well. In addition, the solution also matched the solution data very well. However, when I did predictions, the results did not look good." Could you provide more details? Use l2 regulization to avoid overfitting.
